spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerAutoConfiguration.java

```diff
@@ -121,4 +138,9 @@ public SpannerMutationFactory spannerMutationFactory(
 			SpannerMappingContext spannerMappingContext) {
 		return new SpannerMutationFactoryImpl(spannerConverter, spannerMappingContext);
 	}
+
+	@VisibleForTesting
```

Comment:
We prefer do not provide any test-related stuff in the `main` code.
For the purpose to get access to the class properties there is a `DirectFieldAccessor` to be user from the tests.